src/main/java/com/conveyal/r5/transit/TransferFinder.java

```diff
@@ -181,33 +179,71 @@ public void findTransfers () {
 
 
     /**
-     * Filter down a map from target stop indexes to distances so it only includes those stops that are the
-     * closest on some pattern. This is technically incorrect (think of transfers to a U shaped metro from a bus line
-     * running across the legs of the U, a situation which actually exists in Washington, DC with the
-     * red line and the Q4) but anecdotally it speeds up computation by up to 40 percent. We may want to look into
-     * other ways to optimize transfers (or why the transfers are making routing so much slower) if this turns out to
-     * affect results.
+     * Filters down a map of potential transfer target stops so that, for each pattern, only the closest target stop
+     * is retained; if the closest stop is the source stop itself, the next closest stop is also retained. This
```

Comment:
Based on the code I think this is actually what happens:
```suggestion
     * is retained; if the closest stop is the source stop itself, the next closest stop is retained instead. This
```